**CONFIDENCE INTERVAL**  



**BACKGROUND**

Frequently, testing in manufacturing is done destructively and is costly. Hence, companies use minimal sample sizes, which show the quality of the products. In the present case, a manufacturer of print-head technology faces the requirement of estimating the number of characters that print-heads can print before they fail. Since each test destroys the print-head, only a small sample of 15 units was used

**GIVEN DATA**

Durability of 15 random print-head units (millions of characters):  

1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29


**TASK A: 99% CONFIDENCE INTERVAL WITH SAMPLE STANDARD DEVIATION**



**GOAL** :

If we assume that we don't know the population standard deviation, we can use the sample data to estimate the population mean of durability.



**WHY DO WE CHOOSE T-DISTRIBUTION**

We are using t-distribution because the standard deviation population is not known and our sample size is small (n = 15). Because it takes into consideration the additional uncertainty when estimating the standard deviation from the data, it is more appropriate in these situations than the normal (z) distribution.



**STEPS**:

1. we need to determine the standard deviation and sample mean.

2. finding the t-critical value for 14 degrees of freedom and a 99% confidence level,

3. Calculating the error margin

In [2]:
import numpy as np
from scipy.stats import t

data = [1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]

n = len(data)
mean = np.mean(data)
std_dev = np.std(data, ddof=1)
confidence = 0.99

t_critical = t.ppf((1 + confidence) / 2, df=n-1)
margin_error = t_critical * (std_dev / np.sqrt(n))

ci_lower = mean - margin_error
ci_upper = mean + margin_error

print(f"99% CI: ({ci_lower:.3f}, {ci_upper:.3f})")

99% CI: (1.090, 1.387)


The mean durability of the printing heads is somewhere in the range from 1.090 to 1.387 million characters because of 99% confidence. The t-distribution was applied to get a more precise outcome since the sample size is small and the standard deviation is unknown.

**TASK B: 99% CONFIDENCE INTERVAL WITH KNOWN POPULATION STANDARD DEVIATION**



**GOAL**  

Is to calculate the 99% confidence interval and the population standard deviation is assumed to be 0.2 million characters.

**THE Z-DISTRIBUTION: WHY USE IT? **

As long as the data is normal we can use z-distribution with small sample



**ACTIONS TAKEN**  

1st Utilize the same sample mean as previously  

2nd is for the confidence level of 99% a z-critical value should be taken.

Step 3rd is From the known standard deviation the margin of error is to be calculated

In [3]:
from scipy.stats import norm

pop_std_dev = 0.2
z_critical = norm.ppf((1 + confidence) / 2)

margin_error_z = z_critical * (pop_std_dev / np.sqrt(n))
ci_lower_z = mean - margin_error_z
ci_upper_z = mean + margin_error_z

print(f"99% CI with known σ: ({ci_lower_z:.3f}, {ci_upper_z:.3f})")



99% CI with known σ: (1.106, 1.372)


the standard deviation is calculated so the confidence interval will be tighter than before. This reflects less uncertainty in the estimate as compared to when the standard deviation is derived from the sample.

**Two popular techniques for calculating a population mean from a sample were demonstrated in this assignment**:

1. When the standard deviation is unknown, the t-distribution   

2. When the population standard deviation is known, the z-distribution

Each strategy was selected using the information at hand, and both provided helpful confidence intervals.   

 In quality control, this type of estimation is crucial, particularly when testing is destructive and sample sizes need to be kept small.